modules/local/create_path.nf

```diff
@@ -0,0 +1,25 @@
+process CREATE_PATH {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::coreutils=8.31"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gnu-wget:1.18--h36e9172_9' :
+        'biocontainers/gnu-wget:1.18--h36e9172_9' }"
+
+    input:
+    tuple val(meta), val(accession)
+
+    output:
+    val meta                 , emit: meta
+    path "${meta.id}.txt"    , emit: accession
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix         = task.ext.prefix ?: "${meta.id}"
+    """
+    echo $accession > ${prefix}.txt
```

Comment:
This looks like overkill for a process. I'll double check if this should be replaced.